CHANGELOG.MD

```diff
@@ -0,0 +1,36 @@
+# Change Log
+## 1.1 Release
+### New features
+	- Added Centos 7 build
+	- RPR Render Settings moved to tab in "Render Settings" LOP node for use in Houdini.
+	- Image Gamma supported for GL textures
+	- Add support for alpha compositing render output.  This allows overlaying on the Houdini background and viewport gizmos
+	- Support for UsdTransform2D nodes in shading (Only single level, not nested)
+	- Per object subdivision.  Also added a HDA RPR Mesh Properties Node for specifying subdivision level in Houdini 
+	- Support of doublesided meshes
+	- Support of HdDisplayStyle::displacementEnabled attributes
+	- Camera FStop, Focal Length, Aperture, Focus Distance and Clipping Range is supported on cameras
+	- Support for tapered width curves (linear curves only)
+	- Add debugging environment variable.  Set HD_RPR_ERROR_OUTPUT_FILE=some_file.txt to dump USD errors when rendering with hdRPR in Houdini.
+	
+
+### Fixes
+	- Fixed finding Hybrid.so on Ubuntu
+	- Fixed teardown of Denoiser filter when disabled
+	- Fixed material attachment to instances in hybrid modes
```

Comment:
I've pointed to 3 lines actually, not to only "Fixed AOV creation in CPU rendering" line
Github highlighting is confusing here

Reply:
got it.